File: StudentCard.Application/Students/Dtos/PersonLotDtos/PersonDoctoralDtos/DoctoralSemesterDto.cs
using StudentCard.Application.Students.Dtos.PersonLotDtos.BaseDtos;
using StudentCard.Data.Rdpzsd.Enums;
using System;

namespace StudentCard.Application.Students.Dtos.PersonLotDtos.PersonDoctoralDtos
{
    public class DoctoralSemesterDto : BaseSemesterDto
    {
        public DateTime ProtocolDate { get; set; }
        public string ProtocolNumber { get; set; }
        public YearType YearType { get; set; }
    }
}

File: WebApplication/src/modules/student/dtos/person-lot.dtos/person-doctoral.dtos/doctoral-semester.dto.ts
import { YearType } from "src/modules/student/enums/year-type.enum";
import { BaseSemesterDto } from "../base.dtos/base-semester.dto";

export class DoctoralSemesterDto extends BaseSemesterDto {
  protocolDate: Date;
  protocolNumber: string;
  yearType: YearType;
}